pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -537,6 +547,7 @@ func (br *OVSBridge) createPort(name, ifName, ifType string, ofPortRequest int32
 		Type:          ifType,
 		OFPortRequest: ofPortRequest,
 		Options:       optionMap,
+		ExternalIDs:   helpers.MakeOVSDBMap(map[string]interface{}{AntreaIFTypeKey: antreaType}),
```

Comment:
Why we store some information in port externalIDs but some other in interface externalIDs?

Reply:
I want to leverage port externalIDs for variable key-value maps, but use interface external_ids for the "static" fields of `interfacestore.InterfaceConfig` which should be always configured for each Antrea interface

Reply:
what's the benefit of storing them separately? 
@jianjuns do you have any opinions?

Reply:
I also feel all in port data is better, if the format and lifecycle are the same. I wanted to ask but forgot.

BTW, to keep ovsdb interface clean (which has no knowledge of Antrea today), probably antreaType should be an opaque K/V argument of createPort() too.

Reply:
updated.